api/v1alpha3/cloudstackcluster_webhook.go

```diff
@@ -62,8 +62,8 @@ func (r *CloudStackCluster) ValidateCreate() error {
 		errorList = append(errorList, field.Forbidden(field.NewPath("spec", "identityRef", "kind"), "must be a Secret"))
 	}
 
-	if (r.Spec.Account == "") != (r.Spec.Domain == "") {
-		errorList = append(errorList, field.Required(field.NewPath("spec", "account"), "account and domain must be specified together"))
+	if (r.Spec.Account != "") && (r.Spec.Domain == "") {
```

Comment:
Why the change?  The original construct is surprising, but effectively saying that if account == "" is true then domain must == "" must also be true, and if account == "" is false then domain must == "" must also be false.

The changed logic doesn't detect domain provided but account blank.

Reply:
I just read the PR description.  I hadn't understood it quite they way you described it.  Let me think on this a minute...

Reply:
I stand corrected.  The API Ref states that domain must be provided if account is provided, but nothing about the other way around.